recipes/rapidjson/bld.bat

```diff
@@ -0,0 +1,4 @@
+set RAPIDJSON_DIR=%SRC_DIR%
+
+REM Copy headers
+xcopy /S %RAPIDJSON_DIR%\include %LIBRARY_INC%
```

Comment:
Would it be possible to use the `make install` approach on windows too? The benefit is that it will place the `rapidjsonConfig.cmake` and `rapidjsonConfigVersion.cmake`  files in the right configuration folder for cmake under the prefix, allowing cmake to find it when building packages that depend upon rapidjson.